src/app/GitCommands/Git/GitModule.cs

```diff
@@ -4131,9 +4124,20 @@ string GetDateParameter(string param, DateTime? date)
             }
         }
 
-        internal TestAccessor GetTestAccessor() => new(this);
+        private void SetGitSetting(GitSettingLevel settingLevel, string setting, string? value, bool append = false)
+        {
+            Commands.SetGitSetting(GitExecutable, settingLevel, setting, value, append);
+
+            EffectiveGitConfigSettings.Invalidate();
+            if (settingLevel == GitSettingLevel.Local)
+            {
+                LocalGitConfigSettings.Invalidate();
+            }
+        }
+
+        public TestAccessor GetTestAccessor() => new(this);
 
-        internal readonly struct TestAccessor
+        public readonly struct TestAccessor
```

Comment:
Why?

Reply:
because otherwise, `CommonTestUtils.GitModuleTestHelper` cannot access it

---

Test coverage is high IMO. This PR is "just" a refactoring. There are many tests, particularly for the most affected functionality "management of remotes".

Reply:
> because otherwise, `CommonTestUtils.GitModuleTestHelper` cannot access it

Is it IVT'ed to CommonTestUtils?